src/steps/Microsoft.DotNet.UpgradeAssistant.Steps.ProjectFormat/TryConvertOptions.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.IO;
+using Microsoft.DotNet.UpgradeAssistant.Extensions;
+using Microsoft.DotNet.UpgradeAssistant.Steps.ProjectFormat;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Options;
+
+namespace Microsoft.DotNet.UpgradeAssistant
+{
+    public class TryConvertOptions
```

Comment:
As mentioned above, ideally this could just expand the existing [TryConvertProjectConverterStepOptions](https://github.com/dotnet/upgrade-assistant/blob/main/src/steps/Microsoft.DotNet.UpgradeAssistant.Steps.ProjectFormat/TryConvertProjectConverterStepOptions.cs) instead of adding a new options type.